src/core/hle/service/http/http.h

```diff
@@ -22,14 +22,14 @@ typedef u32 ContextHandle;
 
 /// HTTP operation that will be performed by the request (API-exposed).
 enum class RequestType : u32 {
-    NONE     = 0,
-    GET      = 1,
-    POST     = 2,
-    HEAD     = 3,
-    PUT      = 4,
-    DELETE   = 5,
-    POST_ALT = 6,
-    PUT_ALT  = 7
+    None     = 0,
+    Get      = 1,
+    Post     = 2,
+    Head     = 3,
```

Comment:
@archshift DELETE is defined in winnt.h, from winsock2.h, which prevents it from building on windows. I made them all lowercase since I had to make it lowercase for DELETE.